paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziCallback.kt

```diff
@@ -97,7 +98,15 @@ internal class PaparazziCallback(
 
   override fun resolveResourceId(id: Int): ResourceReference? = projectResources[id]
 
-  override fun getOrGenerateResourceId(resource: ResourceReference): Int = resources[resource] ?: 0
+  override fun getOrGenerateResourceId(resource: ResourceReference): Int {
+    // Workaround: We load our resource map from fields in R.class, which are named using Java
+    // class conventions.  Therefore, we need to similarly transform style naming conventions
+    // that contain periods (e.g., Widget.AppCompat.TextView) to avoid false lookup misses.
```

Comment:
so Widget.AppCompat.Textview becomes `Widget_AppCompat_TextView`?

Seems odd, but if its needed then 🤷

Reply:
yea, cuz we use R.class to bootstrap the resource map.  Eventually, we'll prob use another seed, which means this likely would go away/be replaced.